hack/install-etcd.sh

```diff
@@ -25,4 +25,6 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+FOUND="$(echo $PATH | sed 's/:/\n/g' | grep -q "^${KUBE_ROOT}/third_party/etcd$" || true)"
```

Comment:
should we check the version?

Reply:
The point was only to tell users that etcd is not in their path.  We'll call the install function either way.